packages/design-system/src/components/N8nSquareButton/SquareButton.vue

```diff
@@ -0,0 +1,48 @@
+<template functional>
+	<button :class="$style.button" @click="(e) => listeners.click && listeners.click(e)">
+		<span :class="$style.text" v-text="props.label" />
+	</button>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+export default Vue.extend({
+	name: 'n8n-square-button',
+	props: {
+		label: {
+			type: String,
+		},
+	},
+});
+</script>
+
+<style lang="scss" module>
+.button {
+	width: 28px;
+	height: 29px;
+	border-radius: 4px;
```

Comment:
there's a border radius css variable we are using to standarize those https://n8n-io.github.io/n8n-storybook/?path=/story/styleguide-border--border-radius

Reply:
fixed